atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/Sequence.java

```diff
@@ -30,15 +30,19 @@
 @JsonSerialize(as = ImmutableSequence.class)
 @JsonDeserialize(as = ImmutableSequence.class)
 public interface Sequence extends Comparable<Sequence> {
+
+    Comparator<Sequence> SEQUENCE_COMPARATOR = Comparator.comparingLong(Sequence::value);
+
     @JsonProperty("sequence")
+    @Value.Parameter
```

Comment:
nit: When doing this, I typically like to use `@Value.Immutable(builder = false)` to avoid generating the builder altogether

Reply:
I’ll do a quick pass to update the one arg immutables once I’m back on laptop.